rundeckapp/grails-spa/packages/ui-trellis/src/app/components/home/HomeBrowserItem.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="project_list_item" :key="`project${index}`" :data-project="project.name">
+    <div class="row row-hover row-border-top">
+      <div class="col-sm-6 col-md-8">
+        <ui-socket location="project-favorite" section="home" :socket-data="{projectName: project.name}" />
+        <a
+            :href="`/?project=${project.name}`"
```

Comment:
This absolute url will not work in cases where the app is running with a specific context path, such as `/rundeck`.  The URL should be built from the `getRundeckContext().rdBase` which is the base URL for the server,